### ISSUE TITLE
Add suggestion to the readme of using WhatsApp own non-intuitive UI for that

### ISSUE THREAD

**Issue by qgustavor** [theolm/WhatsAppNoContact#41]
### Describe your suggested feature

Just add to the readme something like this:

> If you are looking for this application just for a single-use case or just want to avoid installing a new application, you can use the WhatsApp's own non-intuitive UI for that:
> 1. Open WhatsApp
> 1. Press the button to start a new chat (the one with a + at the top right of the screen on iOS or in the bottom right on Android)
> 1. Press the search button
> 1. Type or paste the telephone number (**including** the country code)
> 1. Select the contact and start chatting
>
> But, of course, you will be missing another functions from this app, like automatically prefilling with a default country code of your choosing and the quick launch tile.

### Other details

Before I knew the existence of this non-intuitive UI, I used to use the WhatsApp API for that too. In fact, I lost the count of how many people I saw having this issue. They could have integrated to their main search view, not just the only after the start new chat view (which most people never use). Also, since they require the country code without explicitly saying it (unlike this app) it makes this feature really hard to discover.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by JonnyTech** [comment]
@qgustavor your suggestion is no good.

> Press the button to start a new chat (the one with a + at the top right of the screen on iOS or in the bottom right on Android)

This brings up the following dialog:

![o4r4w6r4jknmudhx](https://github.com/user-attachments/assets/c018966f-effe-421b-b8bb-6dcef25f320f)

Select `Not now` and you cannot enter a phone number.

Select `Settings` and allow permissions... **BANG!** WhatsApp, Facebook, Meta and whoever they deem appropriate will immediately know ALL of your contacts.

Please understand that the reason for the existence of this app is to prevent the leaking of our private information.

**Comment by JonnyTech** [comment]
@qgustavor agreed, one cannot prevent others sharing ones details. But weird or not, many of us do value somewhat limited and restricted privacy. Your comments are valid and duly noted, nevertheless it is a personal choice.

**Comment by theolm** [comment]
@qgustavor Thanks for opening the issue. I understand that the documentation may be lacking information. I have no objection to updating it. Please open a PR with the changes that you see fit, I'll be more than happy to update it.

@JonnyTech Thanks for helping here. You are right, the app, besides adding more functionality, also respects the privacy of the user since it requires no special permission to be used (the app doesn't even request internet access).

I also plan to add more functionalities in the future like a history of chats opened and maybe a widget with the saved numbers.
